yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -316,7 +316,9 @@ private[spark] class Client(
         destName: Option[String] = None,
         targetDir: Option[String] = None,
         appMasterOnly: Boolean = false): (Boolean, String) = {
+      logDebug("Current Path=" + path)
       val localURI = new URI(path.trim())
+      logDebug("Current Path as URI=" + localURI)
```

Comment:
These should just be removed; at best use string interpolation but probably worth removing.